### ISSUE TITLE
Fine, I've cloned the branch and trained.... then, I did not see the promised results..

### ISSUE THREAD

**Issue by Heermosi** [bendangnuksung/Image-OutPainting#4]
Fine, author. 
May you provide some details on the training for help?
I've got a total black outpainting for the images for the images after 100+ epochs.
I'm using python 3.5
keras 2.2.0
keras-contrib 2.0.8 (the version it takes were 0.0.2 in coding)
and etc
Fine, I though the etc would not affect the result greatly...
Tensorflow 1.8.0
Cuda 9.1
Cudnn 7.0.5

Thank you for reading.

**Comment by caoxu915683474** [comment]
I change the model with the suggestion in WGAN.
I remove sigmoid function of the final layer in discriminator, and use RMSprop optimizer.
I guess it will be helpful.
